### PR TITLE
Restructured docs navigation tree

### PR DESCRIPTION
**Pull request by zevi-r**
Updating the preevy docs

**[Is this a bugfix/feature/doc-improvement?]**
## This is a
- [x] Doc improvement

### CODE REVIEW

File: site/docs/benefits.md
Comment (royra) [inline site/docs/benefits.md:1]: was this file deleted, renamed, split? where should the old link redirect to?

File: site/docs/ci-integration/index.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: Integrating Preevy with your CI
+parent: /ci-integration
+---
+
+## **Automatically create preview environments from your CI Workflow**
+
+Preevy is designed to be easily run in CI workflows, such as [GH Actions](https://github.com/features/actions), [Circle CI](https://circleci.com/) and others. This automatically creates shareable preview environments for every upcoming pull request and code change.
+
+By sharing [profiles](https://preevy.dev/intro/under-the-hood#profile-configuration), Preevy is able to easily deploy to the same VM when new code is pushed to some branch.
```
Comment (royra) [inline site/docs/ci-integration/index.md:10]: 1. the "sharing profiles" feature is not related to deploying to the same VM.
2. "sharing the profile" isn't really sharing - it just means the profile is saved on a cloud storage to be accessible from CI.
3. remote storage of the profile does make it easier to run in CI.

File: site/docs/cloud-providers/index.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+title: Supported Cloud Providers
+parent: /cloud-providers
+---
+
+## **Self-hosted deploy previews in your cloud or Kubernetes cluster**
+
+
+Preevy allows you to easily provision preview environments for your Docker Compose applications to the cloud provider or Kubernetes cluster you're using:
+   - For AWS: use `aws configure`. See [AWS lightsail credentials configurations](/drivers/aws-lightsail#credentials-configuration).
```
Comment (royra) [inline site/docs/cloud-providers/index.md:10]: IMO these instructions belong in the driver doc.

File: site/docs/faq.md [hunk 3]
```diff
@@ -1,4 +1,50 @@
...
-sidebar_position: 6
----
\ No newline at end of file
+sidebar_position: 15
+---
+
+## Common questions about using Preevy
+
+<details>
+  <summary>What is the difference between the Livecycle Docker Extension and Ngrok?</summary>
```
Comment (royra) [inline site/docs/faq.md:9]: This is right under "Common question about using *Preevy*" but doesn't relate to Preevy at all.

Maybe have a separate section about the extension?

File: site/docs/faq.md [hunk 4]
```diff
@@ -1,4 +1,50 @@
...
+  The Livecycle Docker Extension is integrated with Docker, and provides a smoother experience for Docker users.  
+  Consistent URLs, private environments, organizations and Google/Github authentication are supported out of the box.  
+  Livecycle dashboard provides debugging capabilities that includes log inspection, shell access, and container inspection.  
+  Lastly, integration with the Preevy CLI allows you to create preview environments for Pull Requests - remote ephemeral environments that can be used when your development machine is offline.
+</details>
+
+<details>
+  <summary>Which frameworks and languages does Livecycle support?</summary>
+  
+  Livecycle is language and framework agnostic. It works with anything that runs in a Docker container.
```
Comment (royra) [inline site/docs/faq.md:20]:
```suggestion
  <summary>Which frameworks and languages does Preevy support?</summary>
  
  Preevy is language and framework agnostic. It works with anything that runs in a Docker container.
```

File: site/docs/intro/index.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+---
+title: About Preevy
+parent: /intro
+---
+
+## Preevy - Easily create and manage ephemeral environments
+
+Preevy is an open source tool from Livecycle designed to easily provision, manage, and expose ephemeral environments for containerized applications in the cloud.
```
Comment (royra) [inline site/docs/intro/index.md:8]: nit: capitalization 

```suggestion
Preevy is an Open Source tool from Livecycle designed to easily provision, manage, and expose ephemeral environments for containerized applications in the cloud.
```

File: site/docs/security.md
Comment (royra) [inline site/docs/security.md:1]: Is this copy-pasted directly from the same section of the repo README?

File: site/docs/support.md
Comment (royra) [inline site/docs/support.md:1]: Copy pasted?

File: site/docs/telemetry.md
Comment (royra) [inline site/docs/telemetry.md:1]: Copy pasted?

File: site/docs/welcome.mdx [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+    by <a href="https://livecycle.io">Livecycle</a>
+  </p>
+</div>
+
+<span style={{ fontSize: "x-large" }}>
+  Easily Provision <span style={{ fontStyle: "italic" }}>preview environments </span>{" "}
+  for Docker Compose applications 
+</span>
+
+Preevy is an open source tool from Livecycle designed to easily provision, manage, and expose ephemeral environments for containerized applications in the cloud.
```
Comment (royra) [inline site/docs/welcome.mdx:26]: is this copy-pasted from the intro?

File: site/docs/cloud-providers/kube-pod.md [hunk 7]
```diff
@@ -1,6 +1,6 @@
 ---
 sidebar_position: 4
-title: Kubernetes single Pod driver
+title: Deploy to Kubernetes Single Pod
```
Comment (royra) [inline site/docs/cloud-providers/kube-pod.md:3]:
```suggestion
title: Deploy to Kubernetes
```

File: site/docs/cloud-providers/index.md (hunk 2 above)
Comment (zevi-r) [reply site/docs/cloud-providers/index.md]: I believe this is the "driver doc". We just renamed it, per our discussion

File: site/docs/faq.md (hunk 3 above)
Comment (zevi-r) [reply site/docs/faq.md:9]: Agreed. We need to update the FAQ content with more meaningful content This is just a start/placeholder.

File: site/docs/faq.md (hunk 4 above)
Comment (zevi-r) [reply site/docs/faq.md:20]: Same comment as above.

File: site/docs/security.md
Comment (zevi-r) [reply site/docs/security.md:1]: Yes. I was looking for meaningful text. Feel free to modify

File: site/docs/support.md
Comment (zevi-r) [reply site/docs/support.md:1]: These are links to our support/content channels.

File: site/docs/telemetry.md
Comment (zevi-r) [reply site/docs/telemetry.md:1]: Yes. I was looking for something meaningful. Feel free to modify

File: site/docs/welcome.mdx (hunk 6 above)
Comment (zevi-r) [reply site/docs/welcome.mdx:26]: Yes, to an extent

File: site/docs/cloud-providers/index.md (hunk 2 above)
Comment (royra) [reply site/docs/cloud-providers/index.md]: I meant that this paragraph doesn't belong in the index but rather in each driver's specific doc.

File: site/docs/ci-integration/github-plugin.md
[the remark this answers is not in the document]
Comment (zevi-r) [reply site/docs/ci-integration/github-plugin.md:1]: Is this not the last step in a standard CI integration? Where else should we include it?

File: site/docs/cloud-providers/aws-lightsail.md
[the remark this answers is not in the document]
Comment (zevi-r) [reply site/docs/cloud-providers/aws-lightsail.md:1]: Thats fine. I can go with that

File: site/docs/faq.md (hunk 3 above)
Comment (royra) [reply site/docs/faq.md:9]: OK, so how about we separate the FAQs of Preevy, extension and Livecycle site to different docs, with the first being a placeholder?

File: site/docs/benefits.md
Comment (royra) [reply site/docs/benefits.md:1]: redirect to preevy.dev root

File: site/docs/ci-integration/index.md (hunk 1 above)
Comment (royra) [reply site/docs/ci-integration/index.md:10]: I will fix

File: site/docs/cloud-providers/index.md (hunk 2 above)
Comment (royra) [reply site/docs/cloud-providers/index.md]: zevi will fix

File: site/docs/faq.md (hunk 3 above)
Comment (royra) [reply site/docs/faq.md:9]: zevi

File: site/docs/welcome.mdx (hunk 6 above)
Comment (royra) [reply site/docs/welcome.mdx:26]: leaving it

**Pull request merged**